views/administration/school.hbs

```diff
@@ -48,18 +48,18 @@
 							Klassen funktioniert, stellt die Schul-Cloud eine Transferphase zur Verfügung. So ist der
 							Ablauf während eines Schuljahreswechsels:
 						</p>
-						<p class="font-weight-bold">Während des Schuljahres</p>
+						<p class="font-weight-bold">{{$t "administration.label.duringTheSchoolYear" }}</p>
```

Comment:
```suggestion
						<p class="font-weight-bold">{{$t "administration.school.label.duringTheSchoolYear" }}</p>
```
please check the actual pages where code is used. Every single key should reflect that.